### DISCUSSION TITLE
tutorials/graphos-shipping-supergraph/03-field-deprecation

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#60] [course feedback]
# tutorials/graphos-shipping-supergraph/03-field-deprecation

https://www.apollographql.com/tutorials/graphos-shipping-supergraph/03-field-deprecation

**Comment by asigithub** [comment]
If now we fill in the detailedDescription field, then it will be overwritten by the text field.

**Reply by mabuyo** [reply]
Hey there @asigithub, can you clarify what you mean by "overwritten"?
